storage/mysql/map_storage_test.go

```diff
@@ -372,6 +372,81 @@ func TestMapSetGetMultipleRevisions(t *testing.T) {
 	}
 }
 
+func TestMapSetGetBigBatch(t *testing.T) {
```

Comment:
This test won't reproduce the problem because it sits in the `GetMerkleNodes` call.

Reply:
Yup, I wrote this one before the latest finding with the preloader. Figured it was still worth having anyway.